core/src/main/java/tech/tablesaw/api/DoubleColumn.java

```diff
@@ -438,7 +438,7 @@ public IntComparator rowComparator() {
     }
 
     @Override
-    public double get(final int index) {
+    public Double get(final int index) {
```

Comment:
I think I preferred this returning `double` and having `getObject` to get the `Double`. We want to use `double` most often to avoid boxing,which is why I think it's a good default

Reply:
That's true but the other columns worked the other way String, Date, Time, etc. all returned the Object when we call get(). It feels like overkill to have them all also implement getObject() when double is the only exception.